orttraining/orttraining/training_api/training_session.cc

```diff
@@ -115,11 +113,11 @@ Status TrainingSession::CopyBufferToParameters(OrtValue& parameters_buffer, cons
   return module_->CopyBufferToParameters(parameters_buffer, trainable_only);
 }
 
+#if !defined(ORT_MINIMAL_BUILD)
 Status TrainingSession::ExportModelForInferencing(const std::string& inference_model_path,
```

Comment:
this is a todo right?

Reply:
Yes, this is a TODO. I will add a comment.